### PR TITLE
add components for PR 

### PR DESCRIPTION
**Pull request by supreeti**
- [ ] Create a new directory called component.
- [ ] Create a new calculator.js file and create a React component.
- [ ] Use functional components and one child component.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -37,3 +40,77 @@
...
+}
+
+.z {
+  grid-column-start: 1;
+  grid-column-end: 3;
+}
+
+.keypad .btn-Operator:hover {
+  background-color: orangered;
+}
```
Comment (divinecharlotte) [inline src/App.css]: - [ ] kindly consider making sure that your design follows the design snippet you were given in the project requirement to make your design implementation an exact duplicate of the given one  check the images below for more clarrification
- your design 👍 
Screen Shot 2023-06-03 at 08 47 16

- given design to implement 👇 
Screen Shot 2023-06-03 at 08 51 59

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @supreeti       , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- descriptive readme 👍 
- Linters are passing 👍 
 




 

## Required Changes ♻️

- [ ] kindly consider making sure all the boilerplate that got created by the CRA (*texts*, *images*, and *styles*)are deleted. to make your project clean and related 



_Check the comments under the review._
## Optional suggestions

- 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,82 @@
...
+  padding: 3rem 0;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+header h2 {
+  text-align: center;
 }
```
Comment (abdulhamiid) [inline src/App.css]: Your design (It is too small and does not match the given template) 🛠

image

As requested by the previous reviewer, please, create a React component that matches this design: 👀 ✔️ 

image

The Calculator.js component should match the given design: 💪

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @supreeti 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

almost-there

## Highlights 🌟

- ✔️ Use React components.
- ✔️ Linters checks are passing
- ✔️ Use Git Flow

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @supreeti 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights


✔️ Clean Codebase
✔️ You really did great with the UI
✔️ Well done fixing all the linters errors

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**